backend/endpoints/rooms.py

```diff
@@ -41,19 +41,22 @@ async def create_room(
         HTTP_200_OK (room already exist): {room_id}
```

Comment:
Please, kindly follow the Google Python Style Guide for the docstring format. @AI-fae

Reply:
done